sample/strategy.py

```
@@ -127,49 +181,71 @@ def strategy(exit_option: ExitOption, buffer: float, predict_file: str) -> List[
-                })
-                trade.exit_trade(exit_ts=ser.index[-1], exit_price=ser.mean().round(ROUND_6), exit_option=EXIT_EOM, exit_signal_logs=f'EOM on {ser.index[-1].date()}')
+                trade.foo.update({K_EXIT: EXIT_PRICING, K_PRICING_DATE_LIST: ser.index})
+                trade.exit_trade(
+                    exit_ts=ser.index[-1],
+                    exit_price=ser.mean().round(ROUND_6),
+                    exit_option=EXIT_EOM,
+                    exit_signal_logs=f"EOM on {ser.index[-1].date()}",
+                )
             else:
-                assert ValueError(f'{exit_option} is not a valid exit_option')
+                assert ValueError(f"{exit_option} is not a valid exit_option")
```

Ruff (0.12.2)

Use of `assert` detected

(S101)